### PR TITLE
Add third party connectors `source-kelkoo` and `source-criteo`

### PR DESCRIPTION
**Pull request by jshearer**
NOTE: These depend on `ghcr.io/estuary/gjbakerross/source-criteo`/`source-kelkoo` which are built from dockerfiles in the upstream repos using the following command:

```
docker buildx build --no-cache --platform linux/amd64 https://github.com/gjbakerross/airbyte-source-criteo.git -t ghcr.io/estuary/gjbakerross/source-criteo:dev --push
```

### CODE REVIEW

File: airbyte-integrations/third-party/gjbakerross/airbyte-source-criteo/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ARG AIRBYTE_TO_FLOW_TAG="dev"
+FROM ghcr.io/estuary/airbyte-to-flow:$AIRBYTE_TO_FLOW_TAG as airbyte-to-flow
+
+FROM ghcr.io/estuary/gjbakerross/source-criteo:dev
+
+COPY *.json ./
+# COPY streams/* ./streams/
```
Comment (mdibaiee) [inline airbyte-integrations/third-party/gjbakerross/airbyte-source-criteo/Dockerfile:7]: I would recommend keeping this uncommented. It won't have any negative effect, but will prevent mistakes where we add stream patches without effect, and we have to figure out why the stream patches are not working

**Comment by mdibaiee** [comment]
@jshearer I think it would be good to have a file (READMe maybe?) that tells you how to build the base image for these connectors

**Comment by mdibaiee** [comment]
A few things to check with these connectors:
- [ ] [Check the rendering of the form in the UI](https://github.com/estuary/airbyte#checking-the-rendering-of-the-form-on-the-ui)
- [ ] [Run the connectors and see if they work, and if their document schemas work](https://github.com/estuary/airbyte#checking-the-schema-of-different-connector-streams-collection-schemas)

File: airbyte-integrations/third-party/gjbakerross/airbyte-source-criteo/Dockerfile (hunk 1 above)
Comment (jshearer) [reply airbyte-integrations/third-party/gjbakerross/airbyte-source-criteo/Dockerfile:7]: Yeah so I had originally tried that and it didn't work, but I realized that I was building from the root of the project, not from the connector directory. I uncommented these lines!

**Comment by jshearer** [comment]
@mdibaiee I added a `README.md`. WRT validating that the connectors work, the customer who built these is using them, and they're producing data. He went back and forth with dave & I a couple of times building updates/changing schemas and all seems to be good now.

**Review by mdibaiee — APPROVED** [review]
thanks!

**Pull request merged**